web/src/app/donate/PayWithPayPalComponent.js

```diff
@@ -1,10 +1,17 @@
+import paypal from 'paypal-checkout';
+import PropTypes from 'prop-types';
 import React, { Component } from 'react';
 import RestClient from '../RestClient';
-import paypal from 'paypal-checkout';
 
 class PayWithPayPalComponent extends Component {
   // TODO: clean up code
   componentDidMount() {
+
+    const streamerPaypalEmail = this.props.streamerPaypalEmail;
+    const donationAmount = this.props.amount;
+    const bounty = this.props.bounty;
+    const username = this.props.username;
+
     paypal.Button.render({
 
       env: 'sandbox', // sandbox | production
```

Comment:
I think a spacing is off here. Also we should put the paypal clientID as a named constant or even move it out to a constant file.

Reply:
talked offline. made the call to make a backlog task for cleaning up this PayPal Component instead of doing the work now